addons/packages/antrea/1.7.1-p1/bundle/config/kapp-config.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: kapp.k14s.io/v1alpha1
+kind: Config
+rebaseRules:
+- path: [data]
+  type: copy
+  sources: [existing, new]
+  resourceMatchers:
+  - kindNamespaceNameMatcher: {kind: ConfigMap, namespace: kube-system, name: antrea-ca}
+  - kindNamespaceNameMatcher: {kind: Secret, namespace: vmware-system-antrea, name: nsx-cert}
+  - kindNamespaceNameMatcher: {kind: ConfigMap, namespace: vmware-system-antrea, name: bootstrap-config}
+  - kindNamespaceNameMatcher: {kind: ConfigMap, namespace: vmware-system-antrea, name: nsx-cert}
+
+- path: [spec, caBundle]
+  type: copy
+  sources: [existing, new]
+  resourceMatchers:
+  - anyMatcher:
+      matchers:
+      - kindNamespaceNameMatcher: {kind: APIService, namespace: kube-system, name: v1alpha1.stats.antrea.io}
+      - kindNamespaceNameMatcher: {kind: APIService, namespace: kube-system, name: v1alpha1.stats.antrea.tanzu.vmware.com}
```

Comment:
No apiservice changes?

Reply:
I have checked and there is no APIService in interworking. 
Do you have any comments about this?@edwardbadboy

Reply:
Correct. No APIService in interworking.